### DISCUSSION TITLE
Remove useless aria roles

### DISCUSSION THREAD

**Discussion by avetisk** [gpbl/react-day-picker#2305] [Ideas]
As per standard definition/recommendation, elements such as `<button>` or `<th>` should not have `role` attribute which are already implicitly defined.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/columnheader_role#best_practices

**Comment by gpbl** [comment]
Thanks for the suggestion @avetisk. I agree to remove them! We should confirm first that VoiceOver reads them correctly: it should - but better double check.
